starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,80 @@
 // Names and Input
-
+Abi Summers Github abisummers
+Manon Salaun Github MSLN8
+// Paste your code here
+var hacker1 = "Abi"
+console.log ("the drive's name is "+ hacker1 )
+var hacker2= "Manon"
```

Comment:
You can ask the user to enter a name using the prompt command, but you did that perfectly well later with the palindrome question!